remoteoss/json-schema-form
# feat: Initial release

This library was part of [`@remotecom`](https://github.com/remoteoss) internal source code.

As we are open-sourcing it, this is a copy-paste of its sourcecode.

The docs are copied at #2.

## How to review this:
- Check if the `README` looks good
- Check if the `package.json` looks good
- ESLint/Prettier and Husky are working correctly
- The tests are passing (Already connected to the CI using Github actions)
- Changes to `src/` are out of scope as they are copy-paste
